### ISSUE TITLE
use space low performance when get session and release frequently

### ISSUE THREAD

**Issue by HarrisChu** [vesoft-inc/nebula#2879]
version  v2.5.0

in stress test

1. get session
2. run `use sf100;GO 1 STEP FROM 2608 OVER KNOWS`
3. then release the session.

concurrent 100.

latency p90 is 106196 us.
```

     ✓ IsSucceed

     █ setup

     █ teardown

     checks...............: 100.00% ✓ 68138      ✗ 0
     data_received........: 0 B     0 B/s
     data_sent............: 0 B     0 B/s
     iteration_duration...: avg=440.59ms     min=7.86ms med=167.36ms max=12.62s p(90)=1.23s    p(95)=1.93s
     iterations...........: 68138   225.940829/s
     latency..............: avg=73200.452347 min=2625   med=72990    max=165516 p(90)=106196   p(95)=114443.05
     responseTime.........: avg=74139.420764 min=3679   med=73915    max=166501 p(90)=107113.5 p(95)=115384.9
     vus..................: 100     min=0        max=100
     vus_max..............: 100     min=100      max=100
```

profile when running stress test.

```
(root@nebula) [sf100]> profile {use sf100}
Execution succeeded (time spent 81111/87323 us)

Execution Plan (optimize time 27 us)

-----+------------------------+--------------+-----------------------------------------------------+-------------------------------------------
| id | name                   | dependencies | profiling data                                      | operator info                            |
-----+------------------------+--------------+-----------------------------------------------------+-------------------------------------------
|  1 | RegisterSpaceToSession | 0            | ver: 0, rows: 0, execTime: 23us, totalTime: 80843us | outputVar: [                             |
|    |                        |              |                                                     |   {                                      |
|    |                        |              |                                                     |     "colNames": [],                      |
|    |                        |              |                                                     |     "type": "DATASET",                   |
|    |                        |              |                                                     |     "name": "__RegisterSpaceToSession_1" |
|    |                        |              |                                                     |   }                                      |
|    |                        |              |                                                     | ]                                        |
|    |                        |              |                                                     | inputVar:                                |
|    |                        |              |                                                     | space: sf100                             |
-----+------------------------+--------------+-----------------------------------------------------+-------------------------------------------
|  0 | Start                  |              | ver: 0, rows: 0, execTime: 0us, totalTime: 19us     | outputVar: [                             |
|    |                        |              |                                                     |   {                                      |
|    |                        |              |                                                     |     "colNames": [],                      |
|    |                        |              |                                                     |     "type": "DATASET",                   |
|    |                        |              |                                                     |     "name": "__Start_0"                  |
|    |                        |              |                                                     |   }                                      |
|    |                        |              |                                                     | ]                                        |
-----+------------------------+--------------+-----------------------------------------------------+-------------------------------------------

Thu, 16 Sep 2021 10:51:18 CST
```

the meta leader disk io is high

![image](https://user-images.githubusercontent.com/1726587/133557479-1e101fc9-47e8-412a-bfc0-d1742a52cf20.png)

**Comment by Sophie-Xie** [comment]
@CPWstatic discussed this issue with Forest and Harris,  it will be resolved in next version.

**Comment by HarrisChu** [comment]
fixed in v2.6.0

only poor performance when create or delete session frequently
